stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -261,6 +265,41 @@ public func spawnChild(_ args: [String])
 
     // Close the pipe when we're done writing the error.
     close(childToParentPipe.writeFD)
+  } else {
+    close(childToParentPipe.writeFD)
+
+    // Figure out if the child was execve correctly or not.
```

Comment:
This comment doesn't make sense to me.

Reply:
I think I was bending English a little bit too much in these comments. I was using “execve” as a verb. I wanted to say “Figure out if the child’s call to execve was successful or not”. I will change it.